host/src/PlaceholderApp/JitTrace/JitTraceRuntime.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Copied from Host repo.
```

Comment:
Can you add a link for the extra code reference you copied from?

Reply:
edit: I see you added that on line 15, I think it makes more sense up here at the top level

Reply:
Moved it to the top .